docker/deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
+
+sudo apt-get update && sudo apt-get install -y python-pip && sudo pip install awscli
+
+# Get login token and execute login
+$(aws ecr get-login --no-include-email --region $AWS_REGION)
+echo -e $GITLAB_PRIVATE_KEY > .ssh/id_rsa
+chmod 0500 .ssh/id_rsa
+
+echo "Building latest image with solver...";
+docker-compose build --build-arg use_solver=1 stablex
```

Comment:
maybe you want to build stablex-rinkeby?

Reply:
I didn't because the only difference is the .env file which doesn't get build into the image anyways (it only gets loaded when you run the image using docker-compose which on AWS we don't do).

In the long run, we should probably only have one var that indicates which network we want, so I thought we might be able to get rid of stablex-rinkeby and just pass this single parameter via the `-e` flag when running for rinkeby on our local machine.